test/e2e-go/cli/algod/expect/algodTelemetryLocationTest.exp

```diff
@@ -19,6 +19,7 @@ if { [catch {
     #allows script to be run outside of go context
     exec mkdir -p $TEST_PRIMARY_NODE_DIR
 
+    exec goal node start -d $TEST_PRIMARY_NODE_DIR
```

Comment:
I'm looking on this one and I'm wondering why we need to make the call to `goal node stop` ?
Can you try to run it without it to see if the test passes ?